Modules/Scripted/DICOMPlugins/DICOMScalarVolumePlugin.py

```diff
@@ -56,18 +56,45 @@ def settingsPanelEntry(panel, parent):
     by the DICOM module.
     """
     formLayout = qt.QFormLayout(parent)
+
     readersComboBox = qt.QComboBox()
     for approach in DICOMScalarVolumePluginClass.readerApproaches():
       readersComboBox.addItem(approach)
-
-    readersComboBox.toolTip = "Preferred back end.  Archetype was used by default in Slicer before June of 2017.  Change this setting if data that previously loaded stops working (and report an issue)."
-
+    readersComboBox.toolTip = ("Preferred back end.  Archetype was used by default in Slicer before June of 2017."
+      "Change this setting if data that previously loaded stops working (and report an issue).")
     formLayout.addRow("DICOM reader approach:", readersComboBox)
-
     panel.registerProperty(
       "DICOM/ScalarVolume/ReaderApproach", readersComboBox,
       "currentIndex", str(qt.SIGNAL("currentIndexChanged(int)")))
 
+    importFormatsComboBox = ctk.ctkComboBox()
+    importFormatsComboBox.toolTip = ("Enable adding non-linear transform to regularize images acquired irregular geometry:"
+      " non-rectilinear grid (such as tilted gantry CT acquisitions) and non-uniform slice spacing."
+      " If no regularization is applied then image may appear distorted if it was acquired with irregular geometry.")
+    importFormatsComboBox.addItem("default (none)", "default")
+    importFormatsComboBox.addItem("none", "none")
+    importFormatsComboBox.addItem("apply regularization transform", "transform")
+     # in the future additional option, such as "resample" may be added
+    importFormatsComboBox.currentIndex = 0
+    formLayout.addRow("Acquisition geometry regularization:", importFormatsComboBox)
+    panel.registerProperty(
+      "DICOM/ScalarVolume/AcquisitionGeometryRegularization", importFormatsComboBox,
+      "currentUserDataAsString", str(qt.SIGNAL("currentIndexChanged(int)")),
+      "DICOM examination settings", ctk.ctkSettingsPanel.OptionRequireRestart)
+    # DICOM examination settings are cached so we need to restart to make sure changes take effect
+
+    allowLoadingByTimeCheckBox = qt.QCheckBox()
+    allowLoadingByTimeCheckBox.toolTip = ("Offer loading of individual slices or group of slices"
+      " that were acquired at a specific time (content or trigger time)."
+      " If this option is enabled then a large number of loadable items may be displayed in the Advanced section of DICOM browser.")
+    formLayout.addRow("Allow loading subseries by time:", allowLoadingByTimeCheckBox)
+    exitMapper = ctk.ctkBooleanMapper(allowLoadingByTimeCheckBox, "checked", str(qt.SIGNAL("toggled(bool)")))
```

Comment:
I would rename the mapper instance variable to allowLoadingByTimeBoolMapper or similar.

Reply:
Good catch, I'll update it.